### PR TITLE
Implemeneted Birth date Detection

### PR DESCRIPTION
**Pull request by tolanism**

### CODE REVIEW

**Review by claesmk — CHANGES_REQUESTED** [review]
Your code is not actually detecting anything.  If you do not want to implement the custom recognizer discussed in your issue, you should at least look for the DATE_TIME entity which would at least indicate a date is present.

**Comment by tolanism** [comment]
> Your code is not actually detecting anything. If you do not want to implement the custom recognizer discussed in your issue, you should at least look for the DATE_TIME entity which would at least indicate a date is present.

Is that why it's failing the pytest?

**Comment by claesmk** [comment]
@tolanism I pushed some changes that fix your indentation issue, duplicated function, and added a DATE_TIME example.  You should be able to sync (push/pull) and see these changes in your codespace.

**Review by claesmk — DISMISSED** [review]
Positive and negative cases for DATE_TIME detection are working. If you want to merge this, you can, or you can try to implement the custom recognizers suggested in your issue

**Comment by tolanism** [comment]
> Positive and negative cases for DATE_TIME detection are working. If you want to merge this, you can, or you can try to implement the custom recognizers suggested in your issue

ok! Thank you. I've been reading [this] you suggested earlier (https://microsoft.github.io/presidio/samples/python/customizing_presidio_analyzer/#example-6-using-context-words)
to understand the custom recognizers

File: test_team_code_busters.py [hunk 1]
```diff
@@ -3,10 +3,7 @@
 
 
 class TestTeamCodeBusters(unittest.TestCase):
-    def test_aggie_pride(self):
-        """Test to make sure the Aggie Pride function works"""
-        self.assertEqual('Aggie Pride - Worldwide', show_aggie_pride())
-
+
```
Comment (claesmk) [inline test_team_code_busters.py:6]: Not sure what you were trying to do here?

**Review by tfortobi — APPROVED** [review]
Looks good to me, good job.

**Review by Lesterconey — DISMISSED** [review]
Looks good.

**Review by Lesterconey — APPROVED** [review]

**Review by chelseavuata — DISMISSED** [review]
Looks good!

**Review by claesmk — APPROVED** [review]
I resolved your merge conflicts and updated birthdate to BIRTHDATE to maintain consistency with other entity names

**Pull request merged**